src/lib/interfaces/api-params.ts

```diff
@@ -134,6 +134,19 @@ export interface PublisherUserAccessCheckParams {
   cross_app?: boolean;
 }
 
+export interface PublisherUserEmailGetParams {
+  mail_log_id: string;
+}
+
+export interface PublisherUserEmailListParams {
+  uid: string;
+  offset: number;
+  include_email_body?: boolean;
+  order_by?: string;
+  order_direction?: string;
```

Comment:
We actually have an [OrderDirection](https://github.com/kodansha/piano-sdk-for-nodejs/blob/c97e7b2c846211e7cd33b556798f16596906966b/src/lib/interfaces/api-params.ts#L1) type so if you could use that here it would be great.